data.sql

```diff
@@ -0,0 +1,10 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, weight_kg, neutered)
+ VALUES (1, 'Agumon', 'Feb 3, 2020', 0, 10.23, true);
```

Comment:
[OPTIONAL]
- [ ] To make your codebase more readable and ready to understand you may also consider using this format for your date,
```sql
VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```
- [ ] To avoid adding the id every time a value is added to the table, consider setting the id to an auto-increment like so while creating your table.
```sql
id int PRIMARY KEY NOT NULL IDENTITY(1,1)
-- In this case to add a value you will only have to do: --
-- VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23'); --
```

Reply:
Updated 😀